revenue/revenueapp/templates/revenueapp/venue_form.html

```diff
@@ -22,7 +22,8 @@
             <li><a href="{% url 'review_create' %}">Review</a></li>
         </ul>
     </div>
-    <form method="post">
+    <h1 id="Review_update_text" >Please create a new venue to review</h1>
+    <form id="Review_form">
```

Comment:
The `method` attribute is necessary for the form to work. Please add it back to this form element, as well.